src/main/java/org/folio/service/export/storage/AWSStorageServiceImpl.java

```diff
@@ -106,10 +113,20 @@ public void storeFile(FileDefinition fileDefinition, String tenantId) {
         multipleFileUpload.waitForCompletion();
       } catch (InterruptedException e) {
         LOGGER.error(e.getMessage());
+        populateErrorLog("S3 connection issue", fileDefinition.getJobExecutionId(), tenantId);
```

Comment:
This is a really rare case when InterruptedException is thrown, usually, it happens when the application gets shutdown. No need to populate errors here